codebundles/k8s-fluxcd-reconcile/.runwhen/generation-rules/k8s-fluxcd-reconcile.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: runwhen.com/v1
+kind: GenerationRules
+spec:
+  generationRules:
+    - resourceTypes:
```

Comment:
update qualifier to cluster level -> annotation on a deployment matching fluxcd crd names